arkanoid/js/ball.js

```diff
@@ -0,0 +1,91 @@
+function Ball(x, y) {
+  this.velocity = 6;
+  this.dx = 0;
+  this.dy = 0;
+  this.x = 440;
+  this.y = 440;
+  this.width = 40;
+  this.height = 40;
+
+  this.start = function () {
+    this.dy = -this.velocity;
+    this.dx = this.getRandomDirection(-this.velocity, this.velocity);
+  };
+
+  this.fly = function () {
+    if (this.dy) {
+      this.y += this.dy;
+    }
+    if (this.dx) {
+      this.x += this.dx;
+    }
+  };
+
+  this.collide = function (element) {
```

Comment:
Naming. There's convention to call boolean-returning variables/methods with "yes or no" question statements.
For instance: "isCollision", "collides" etc.